mathesar_ui/src/systems/table-view/row/RowCell.svelte

```diff
@@ -179,8 +183,13 @@
         Set to <Null />
       </ButtonMenuItem>
       {#if linkedRecordHref}
-        <LinkMenuItem icon={iconLinkToRecordPage} href={linkedRecordHref}>
-          Go To Linked Record
+        <LinkMenuItem icon={iconRecord} href={linkedRecordHref}>
+          Open {$recordSummaries.get(String(column.id))?.get(String(value))}
+        </LinkMenuItem>
+      {/if}
+      {#if fk_table_link}
+        <LinkMenuItem icon={iconTable} href={fk_table_link}>
+          Open {column.name}
```

Comment:
Here (and also in the column header link), we need to make the following changes:

- Show the name of the linked _table_, not the name of the linking column.
- Wrap the table name in an `Identifier` component (effectively making it bold, but also allowing us to further adjust this style globally later as we see fit).
- Append the word "Table" to the label

Currently I see

> Open Patron

![image](https://user-images.githubusercontent.com/42411/215164655-716ab599-c699-4577-b1bd-8dd907227561.png)

Instead, I should see

> Open **Patrons** Table

(Note that in the schema I'm using I have a column named `Patron` which links to a table named `Patrons`.)